src/app/dashboard/Missions/CreateMission.js

```diff
@@ -0,0 +1,489 @@
+    setPickUp({ ...pickUp, [event.target.name]: event.target.value });
+  };
+
+  const handleComment = (event) => {
+    setComment(event.target.value);
+  };
+
+  const handleItems = (event) => {
+    setItems({ ...items, [event.target.name]: event.target.value });
+    console.log(items);
+  };
+
+  // This handles all the food box inputs that are added to food box missions
+  const [foodBoxArray, setFoodBoxArray] = useState([]);
+  function addToFBCount() {
+    setFoodBoxArray([...foodBoxArray, foodBoxArray.length]);
+  }
+
+  return (
+    <Container maxWidth="sm">
+      <Typography className={classes.addMissionContainer} component="div">
+        <div className={classes.title}>
+          <AnnouncementIcon className={classes.icon} />
+          <Typography className={classes.titleHeader}>Add New Mission</Typography>
+        </div>
+        <Divider variant="middle" />
+        <Typography className={classes.inputHeader}>Mission Type</Typography>
+        <FormControl className={classes.form}>
+          <InputLabel
+            variant="outlined"
+            id="missionType"
+            shrink={true}
+            className={classes.inputLabel}
+          >
+            Select Mission Type
+          </InputLabel>
+          <Select
+            labelid="missionType"
+            variant="outlined"
+            className={classes.select}
+            value={missionType}
+            onChange={handleChangeMissionType}
+          >
+            <MenuItem className={classes.menuItem} value="food-box">
+              Food box
+            </MenuItem>
+          </Select>
+        </FormControl>
+        {/* THIS PART SPAWNS BASED ON WHAT MISSION TYPE WAS SELECTED */}
+        {missionType === "food-box" ? (
+          <>
+            <Typography className={classes.inputHeader}>Select Food Box Options</Typography>
+            <FormControl className={classes.form}>
+              {Object.keys(foodBoxArray).map((index) => {
+                const { amount, name } = foodBoxArray[index];
+                return (
+                  <div className={classes.sideBySideSelect} key={index}>
+                    <Select
+                      onChange={handleItems}
+                      name={"item for box " + index}
+                      variant="outlined"
+                      className={classes.select}
+                      value={name}
+                    >
+                      <MenuItem value="Fruits and Veggies">Fruits & Veggies</MenuItem>
+                      <MenuItem value="Non-Perishables">Non-Perishables</MenuItem>
+                      <MenuItem value="Meat">Meat</MenuItem>
+                    </Select>
+                    <TextField
+                      onChange={handleItems}
+                      name={"amount for box " + index}
+                      type="number"
+                      variant="outlined"
+                      className={classes.numSelector}
+                      value={amount}
+                    />
```

Comment:
Will need to consult with design. Design showing this as as select with a dropdown, but the form here is a number input that lets the user enter in negative numbers.

Reply:
Didn't mention we'll want to know what the max # items an org can add with this.

Reply:
I added a min, but fro right now I think a max would be hard to find unless we had some hard data on how Organizations are gonna use our application and for what amount. I suspect it will be less than 100, but I think setting a max would be jumping the gun for now.